target/linux/ipq40xx/files-4.19/arch/arm/boot/dts/qcom-ipq4019-eap2200.dts

```diff
@@ -0,0 +1,282 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq4019.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "EnGenius EAP2200";
+	compatible = "engenius,eap2200";
+
+	keys {
+			compatible = "gpio-keys";
+
+			wps {
+				label = "wps";
+				gpios = <&tlmm 18 GPIO_ACTIVE_LOW>;
+				linux,code = <KEY_WPS_BUTTON>;
+			};
+	};
+
+	aliases {
+		led-boot = &power;
+		led-failsafe = &power;
+		led-running = &power;
+		led-upgrade = &power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		power: orange {
+			label = "eap2200:orange:power";
```

Comment:
About the "orange".

There's a push underway to unify some of the various LED colors.

You can find a list of colors here:
https://github.com/torvalds/linux/blob/master/include/dt-bindings/leds/common.h#L28

The closet match to the color "orange" should be "amber". would this work or do you see a issue here?

Reply:
Thanks for comment. Modified the label and led name.